.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,36 @@
-### What does this PR do?:
+# What does this PR do?
```

Comment:
Personal preference: PR descriptions should start at the second level heading (`##`) and use `###` for subheads; the title size is too large and the PR already gets a title set separately.

Reply:
Good point, I'll change that.